tests/test_processors.py

```diff
@@ -149,7 +159,7 @@ def brand(self):
 
 def test_brand_page():
     class MyProductPage(ProductPage):
-        @field
+        @field(out=[brand_processor])
```

Comment:
why is this needed? Shouldn't subclasses of ProductPage have brand_processor enabled by default?

Reply:
Yes, replaced `ProductPage` with `BasePage` in d2ab672afffdc9aad5a064a8345e663d4af8dd33.

Reply:
Actually it needs to be `Page`, not `BasePage`, because `self.css` is not working otherwise, updated in 30baf8a0ca371d93af1118a7829d8f4e607378a8

Reply:
Looks like code coverage tool is complaining, so in the end I had to remove `brand_processor` as originally suggested. Fixed in aa42e22b7db52fc5ab1a711df44f6180069d6d37